exercise/tasks.py

```diff
@@ -0,0 +1,54 @@
+import logging
+from time import sleep
+
+from celery import current_task
+from django.db.models import Q
+from aplus.celery import app
+from .exercise_models import BaseExercise
+
+logger = logging.getLogger('aplus.exercise')
+
+@app.task
+def regrade_exercises(exerciseid: int, uri_prefix: str, regrade_type: str) -> None:
+    exercise = BaseExercise.objects.get(pk=exerciseid)
+
+    qs = exercise.submissions\
+        .defer("feedback", "submission_data", "grading_data")
```

Comment:
```suggestion
    qs = (exercise.submissions
        .defer("feedback", "assistant_feedback", "submission_data", "grading_data")
    )
```

https://apluslms.github.io/contribute/styleguides/python/#indentation-line-length-and-blank-lines